src/libtest/lib.rs

```diff
@@ -34,10 +34,7 @@
 #![deny(warnings)]
 
 #![feature(asm)]
-#![feature(libc)]
-#![feature(rustc_private)]
 #![feature(set_stdio)]
```

Comment:
We can't get rid of this one yet, since there'd be no way to capture the stderr and stdout of tests.